www/front_src/src/Resources/Details/tabs/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,184 @@
+  labelNoContactGroupsIsConfiguredForThisResource,
+  labelNoContactIsConfiguredForThisResource,
+  labelNotificationStatus,
+} from '../../../translatedLabels';
+
+import { Contact, ContactGroup, NotificationContacts } from './models';
+import Contacts from './Contacts';
+import ContactsLoadingSkeleton from './ContactsLoadingSkeleton';
+import ContactCell from './ContactCell';
+
+const Notifications = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const [notificationContacts, setNotificationContacts] =
+    React.useState<NotificationContacts | null>(null);
+
+  const { sendRequest } = useRequest<NotificationContacts>({
+    request: getData,
+  });
+  const details = useAtomValue(detailsAtom);
+
+  const endpoint = path(['links', 'endpoints', 'notification_policy'], details);
+
+  const loadNotificationContacts = (): void => {
+    sendRequest({ endpoint }).then(setNotificationContacts);
+  };
+
+  const getContactColumns = React.useCallback((contact): JSX.Element => {
+    return (
+      <>
+        <ContactCell paddingLeft={1}>{contact.name}</ContactCell>
+
+        <ContactCell>{contact.alias}</ContactCell>
+      </>
+    );
+  }, []);
+
+  const getContactWithEmailColumns = React.useCallback(
+    (contact: Contact): JSX.Element => {
+      return (
+        <>
+          {getContactColumns(contact)}
+          <ContactCell>{contact.email}</ContactCell>
+        </>
+      );
+    },
+    [],
+  );
+
+  React.useEffect(() => {
+    if (isNil(details)) {
+      return;
+    }
+    loadNotificationContacts();
+  }, [details]);
+
+  const contactHeaders = (
+    <>
+      <Typography sx={{ fontWeight: 'bold', paddingLeft: 1 }}>
+        {t(labelName)}
+      </Typography>
+      <Typography sx={{ fontWeight: 'bold' }}>{t(labelAlias)}</Typography>
+    </>
+  );
+
+  const contactWithEmailHeaders = (
+    <>
+      {contactHeaders}
+      <Typography sx={{ fontWeight: 'bold' }}>{t(labelEmail)}</Typography>
+    </>
+  );
+
+  if (isNil(notificationContacts)) {
+    return <ContactsLoadingSkeleton />;
+  }
+
+  return (
+    <Stack spacing={1}>
+      <Paper>
+        <Stack
+          alignItems="center"
+          direction="row"
+          justifyContent="space-between"
+          padding={1}
+        >
+          <Typography sx={{ fontWeight: 'bold' }}>
+            {t(labelNotificationStatus)}
+          </Typography>
+          {notificationContacts?.is_notification_enabled ? (
+            <NotificationIconActive color="primary" />
+          ) : (
+            <NotificationIconOff color="primary" />
+          )}
+        </Stack>
+      </Paper>
+      {isEmpty(notificationContacts.contacts) ? (
+        <Paper>
+          <Stack alignItems="center" direction="row" padding={1} spacing={0.5}>
+            <PersonIcon color="primary" fontSize="large" />
+            <Typography sx={{ fontWeight: 'bold' }}>
+              {t(labelContacts)}
+            </Typography>
+          </Stack>
+          <Stack padding={1}>
```

Comment:
Is a stack surrouning only one element really necessary?

besides that, I think we should center the text indicating that no contacts are available